modules/lang-painless/src/yamlRestTest/resources/rest-api-spec/test/painless/190_tsdb_downsampling.yml

```diff
@@ -0,0 +1,51 @@
+---
+"Unable to define a runtime field as a time series dimension":
```

Comment:
Should this test be under the `lang-painless` module? I think it should be under the tsdb test (probably `rest-api-spec/test/tsdb/70_dimension_types.yml`) or even the `rest-api-spec/test/tsdb/120_runtime_fields.yml`